packages/create-codes/__tests__/cli.test.ts

```diff
@@ -38,7 +39,7 @@ describe("create-codes cli", () => {
       const results = await exeInteractive(cli, [
         testDir,
         keys.ENTER,
-
+        keys.DOWN,
```

Comment:
[MEMO] The current CLI test is written for when selecting SolidJS. It was easier to keep selecting SolidJS than to modify the test for React.